### ISSUE TITLE
[Frontend] Welcome modal doesn't resize vertically with screen

### ISSUE THREAD

**Issue by s-egge** [electricitymaps/electricitymaps-contrib#6731]
**Describe the bug**
The Welcome modal that is displayed when entering the website for the first time does not scale vertically based on the screen size or allow scrolling down to see the whole modal.

**To Reproduce**
1. Open a private browser (or clear history for https://app.electricitymaps.com/map)
2. Go to https://app.electricitymaps.com/map
3. Make browser window smaller vertically

**Expected behavior**
I expected the Welcome modal to either resize or the page to allow me to scroll so that the rest of the modal would come into view.

**Screenshots**
![image](https://github.com/electricitymaps/electricitymaps-contrib/assets/102624422/c1970f98-2d64-4f8c-8b3e-f1ca845f5549)

**Desktop (if relevant):**
 - OS: Windows 10
 - Browser: Firefox and Chrome

**Comment by PmplCode** [comment]
I agree with the expected behavior, I reckon it should allow scrolling.

**Comment by VIKTORVAV99** [comment]
> I agree with the expected behavior, I reckon it should allow scrolling.

I believe we would prefer if it was centred and shrunken to fit the screen size rather than scrolling (within reasonable limits).

**Comment by madsnedergaard** [comment]
The modal is no longer there, so this is fixed :)
